modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -1018,26 +998,13 @@ m|READ
 |===
 | Description
 a|
-Create a named vector index for the given label, and property with given vector dimensionality, using the given similarity function.
-
-`vectorDimension` must be between `1` and `2048` _inclusively_.
-
-Valid case-insensitive values for the `vectorSimilarityFunction` are:
-
-* `"euclidean"` -- Euclidean similarity, where:
-+
-image::euclidean_similarity_equation.svg["The Euclidean of vector v and vector u is defined as 1 over the quantity 1 plus the square of the l2-norm of vector v subtract vector u, which exists in the bounded set of real numbers between 0 exclusive and 1 inclusive."]
```

Comment:
If we are removing the references to similarity functions here, and the images describing their mathematical representation, the images should also be removed from the repository (as I do not believe they are used elsewhere in `doc-operations`)

```
/modules/ROOT/images/cosine_similarity_equation.svg
/modules/ROOT/images/cosine_similarity_equation.tex
/modules/ROOT/images/euclidean_similarity_equation.svg
/modules/ROOT/images/euclidean_similarity_equation.tex
```